src/pages/projects.js

```diff
@@ -385,7 +389,12 @@ const ProjectDescriptions = () => (
             received a major boost in capabilities and sophistication and closed
             many existing gaps with BCC as a library. It also supports important
             features not available in BCC such as global variables and BPF
-            skeletons.
+            skeletons. <a href="https://github.com/xdp-project/xdp-tools">libxdp</a>
```

Comment:
Use </p></p> between descriptions of projects, as is done on the latest once rebasing.